### ISSUE TITLE
No Database

### ISSUE THREAD

**Issue by Malta112** [vocascan/vocascan-server#98]
No database because i dont know what is needet (Using mysql)

**Expected behavior**
Should be able to execute command to database but for example xyz.roles doesn't exist

**Screenshots**
![image](https://github.com/vocascan/vocascan-server/assets/51021169/d189d78d-b551-4248-b0c5-d291ef185c71)

**Additional context**
There was no step on how to create the database

**Comment by luwol03** [comment]
Hello @Malta112,

can you please explain what you did that you got that error and how you attempted the install? And can you verify if the migrations already run by looking for any tables in your db. Especially there should be a table named `.migrations`, if so please post the records that are in there.

**Comment by Malta112** [comment]
I Just followed the instructions to install the backend and there was never a way to create the database and i started the backend but it created only two strange tables:
![image](https://github.com/vocascan/vocascan-server/assets/51021169/5303c9a7-8611-4a26-9f3d-d9a542d5b674)

**Comment by Malta112** [comment]
should it auto create the tables?? Because that didnt happen

**Comment by Malta112** [comment]
![image](https://github.com/vocascan/vocascan-server/assets/51021169/f102f99d-7267-4729-9fbc-d6522dd8b5f5)
So .migrations is empty
and i used npm

**Comment by luwol03** [comment]
Yes, they should be created automatically on start. Please show the full log if you run the server and please also provide some information how you run vocascan-server.

**Comment by Malta112** [comment]
Im following the installation and running it by "pm2 start vocascan-server -- web"

**Comment by Malta112** [comment]
GNU nano 6.2                                                     vocascan.log.gz                                                               
info: migrating...
Executing (default): SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_NAME = '.migrations' AND TABLE_S>
Executing (default): SHOW INDEX FROM `.migrations`
Executing (default): SELECT `name`, `createdAt`, `updatedAt` FROM `.migrations` AS `.migrations` ORDER BY `.migrations`.`name` ASC;
info: ✓ migrating done in (93.873ms)
Executing (default): SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_NAME = '.seeders' AND TABLE_SCHE>
info: seeding...
Executing (default): SHOW INDEX FROM `.seeders`
Executing (default): SELECT `name`, `createdAt`, `updatedAt` FROM `.seeders` AS `.seeders` ORDER BY `.seeders`.`name` ASC;
info: ✓ seeding done in (9.39ms)
info: Server is running on port 5000.

**Comment by noctera** [comment]
Seems like the migrations are running perfectly. 
@luwol03 might this be a mysql specific problem with any type of unknown name resolving? As far as I know we have never tested MySql explicitly, but I know that there might be some differences in the ORM
